docs/guide/test-fixtures.md

```
@@ -36,6 +36,10 @@ class UserFixture extends ActiveFixture
 }
 ```
 
+> Note: These examples use the `app\tests\fixtures` namespace because the fixture classes are referenced directly
+> by application tests. The `yii fixture` command has its own default namespace, `tests\unit\fixtures`.
+> If your fixture classes use another namespace, configure the command or pass the `--namespace` option.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)